action/src/run.ts

```diff
@@ -163,4 +163,9 @@ export const run = async () => {
     ...context.repo
   });
   await createGithubComment();
+
+  if (numVisualTestFailures > 0) {
+    setFailed('Visual regression differences found while taking a screenshot!');
+    // insert logic to send slack notifcation to PR authors
+  }
```

Comment:
I don't think we actually need the "if" statement since we already know there are visual diffs in this branch of the code